internal/types/sidecarconfig.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 SAP SE or an SAP affiliate company
```

Comment:
I feel that will be quite generic and we lose the idea that this is only for the sidecar
wdyt?

Reply:
we should have a name which associates a semantic meaning. In this case options is too generic. If we add more options tomorrow then we can for sure revisit this. For now i think we can go with suggestion from Shreyas as we in future will look to remove backup-restore as a sidecar. So using backuprestore instead of sidecar will be both semantically correct and also future proof.